packages/addons/service/tvheadend42/source/defaults/config

```diff
@@ -1,13 +1,14 @@
 {
-	"uilevel": 1,
-	"uilevel_nochange": false,
+	"wizard": "hello",
+	"uilevel": 0,
+	"uilevel_nochange": true,
 	"ui_quicktips": true,
 	"cookie_expires": 14,
 	"caclient_ui": true,
 	"epg_compress": true,
 	"prefer_picon": true,
 	"chiconpath": "file:///storage/picons/tvh/%C.png",
 	"chiconscheme": 2,
-	"piconpath": "file:///storage/picons/vdr/",
+	"piconpath": "file:///storage/picons/tvh/",
 	"piconscheme": 1
```

Comment:
An addon would be would be handy indeed :-) Is the subdir actually needed, it's confusing that tvh users should put their picons in the vdr folder? Or should that addon be usable soon?